### ISSUE TITLE
Button is displayed as "pressed" when it has keyboard focus

### ISSUE THREAD

**Issue by sbrunot** [ibm-js/deliteful#230]
When a deliteful/Button widget has keyboard focus, it renders as if it was pressed (same rendering than when pressing a mouse button on it an not releasing it). It should have a normal rendering when it has focus, and only render as pressed when the SPACE key is pressed.
